src/components/draft/CivPanel.tsx

```diff
@@ -121,7 +121,7 @@ class CivPanel extends React.Component<IProps, IState> {
         }
         return (
             <div className={className} onClick={onClickAction}>
-                <div className={'stretchy-background'}/>
+                <div className={'stretchy-background'} style={this.props.userThemecolor ? {backgroundColor: this.props.userThemecolor} : {}}/>
```

Comment:
Would it be enough to calculate a custom `backgroundColor` in this component based on the query string and `civPanelType`, without having to pass down a property through a bunch of its parent components? :thinking:

Reply:
My thinking was that we would allow a few other customization options in the future which would be part of a theme object which can be accessed by the entire draft page.